### PR TITLE
Melhorias readme

### PR DESCRIPTION
**Pull request by stanleygomes**
# PR Details

## Description

Give a good explanation about this PR, including about the files you've changed.

## Related Issue 

Issue Title ([PR_ID](http://github.com/link_for_issue))

## Steps to Reproduce

`Write your steps here:`

1. Sample
2. Of
3. Steps

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/7cdc05a8-9023-4af3-8f04-4b8b63ec51f3/deploy-status)](https://app.netlify.com/sites/juggernaut/deploys)
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
+![Lint and test](https://github.com/SoftboxLab/juggernaut/workflows/Lint%20and%20test/badge.svg)
 
 # Juggernaut
 
-## An unstoppable boilerplate
+##  An unstoppable boilerplate
```
Comment (vilaboim) [inline README.md:7]:
```suggestion
## An unstoppable boilerplate
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/7cdc05a8-9023-4af3-8f04-4b8b63ec51f3/deploy-status)](https://app.netlify.com/sites/juggernaut/deploys)
-[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
+![Lint and test](https://github.com/SoftboxLab/juggernaut/workflows/Lint%20and%20test/badge.svg)
```
Comment (vilaboim) [inline README.md:3]: Esse link me parece estar incorreto.

**Review by vilaboim — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stanleygomes) [reply README.md]: Ajustado.

File: README.md (hunk 2 above)
Comment (stanleygomes) [reply README.md]: Ajustado. Tirei o link do status de workflow de lint and test.

**Review by vilaboim — APPROVED** [review]

**Pull request merged**